.travis.yml

```diff
@@ -1,76 +1,68 @@
-language: python
+language: c
 
-python:
-  - 2.7
+os:
+    - linux
+    - osx
 
-virtualenv:
-     system_site_packages: false
+cache:
+    directories:
+        $HOME/miniconda/pkgs
+        /Users/travis/miniconda3/pkgs
```

Comment:
is it `$HOME` == `/Users/travis`? What's the difference between `miniconda` and `miniconda3`